docs/en/setup/README.md

```diff
@@ -0,0 +1,7 @@
+# Setup
+
+The most important thing in E2E Testing is that it uses a separate configuration file and command to execute.
+If you haven't read the [Module Design](../concepts-and-designs/module-design.md), recommend read this document first.
+
+* [Configuration file](Configuration-File.md)
+* [Run E2E Tests](Run-E2E-Tests.md)
```

Comment:
There is no menu pointing to this, consider to remove?

Reply:
It references by https://github.com/apache/skywalking-infra-e2e/blob/3f38aa880d42d5587a02f4e2e5539e279a270634/docs/README.md